projects/virtual/drom.toml

```diff
@@ -1,5 +1,5 @@
 [project]
-drom-version = "0.1.0"
+# drom-version = "0.1.0"
```

Comment:
I have a doubt, does it mean that the `drom-version` is not specified, or the version of `drom` is added as `drom-version` in the `drom.toml` file ? I think it is important that a value is always written, but I don't really care which version is chosen by default.

Actually, there is also the idea that there is a phase of creation of the project, during which these files are modified by the project creator (and all other files are updated), while when the creation phase is over, only a minimal set of files are updated. We could decide that ending the creation phase requires setting the `drom-version`.

Reply:
In the corresponding `drom`, if no `drom-version` is given, `drom` will use its current version.

As for the creation phase, I dont' know... Personally, I always have to change `drom` stuff during a project lifetime so I don't really see a "creation phase". `drom` should simplifies the programmer's life, not complicate it with more (possibly non standard) processes.